### PR TITLE
Adding information about lxplus installation

### PR DESCRIPTION
**Pull request by antonioiuliano2**
Hi to all you FairShip developers,
today I have been contacted by a colleague who received an error, after trying to launch the full installation on lxplus. Therefore, I consider it better to clearly explain in the README that the full installation should not be done there.
Also, I have suggested the usage of lxplus6, since I noticed the display not working in lxplus7.

Best regards,
Antonio

### CODE REVIEW

**Comment by olantwin** [comment]
I would maybe rephrase the recommendation for lxplus6:

> There is currently a known issue with the event display on CC7. As a workaround, lxplus6 can be used until this issue is resolved.

Do we currenlty know why it doesn't work?

In general we should encourage use of lxplus 7, as lxplus 6 won't be supported forever, and we'll only be able to discover and fix these issues if people use CC7.

**Comment by ThomasRuf** [comment]
on cc7, the system libraries changed since building. Currently SHiPBuild points to
SHiPBuild -> /cvmfs/ship.cern.ch/July2018/SHiPBuild
did anybody tried with /cvmfs/ship.cern.ch/Nov2018/SHiPBuild? 

The full installation on lxplus fails because a normal user will not have enough memory for the compilation. However, I thought this issue is being mitigated when we changed to CMAKE_BUILD_TYPE: "RELEASE".

**Comment by antonioiuliano2** [comment]
Hi,

I have tried, unfortunately I continue to get the same error:

libpng warning: Application built with libpng-1.2.55 but running with 1.5.13
TASImage::GetMask:0: RuntimeWarning: No image

Best regards,
Antonio

Il giorno lun 27 mag 2019 alle ore 11:38 Thomas Ruf <
notifications@github.com> ha scritto:
